src/main/resources/templates/pps/penaltyPaid.html

```diff
@@ -13,16 +13,15 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <h1 class="govuk-heading-xl">
-                    Late filing penalty for
-                    <span class="govuk-heading-xl" id="company-name" th:text="${companyName}"></span>
+                    This penalty has already been paid
                 </h1>
-                <p class="govuk-body-l">
-                    The penalty with a reference of
-                    <span class="govuk-!-font-weight-bold" th:text="${penaltyNumber}"></span>
-                    has been paid. No further action is required.
+                <p class="govuk-body-l"> Penalty
+                    <span class="govuk-!-font-weight-bold" th:text="${penaltyNumber}"></span> has been paid.<p></p>
+                <p class="govuk-body-l">You do not need to do anything else.</p>
+                <p></p><p class="govuk-body-l">If you have any questions, you can <a href="https://www.gov.uk/contact-companies-house" class="govuk-link">contact us</a>.</p>
```

Comment:
Is <p></p> need here?

Reply:
It provides spacing between lines, so need to keep it in